Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +86,18 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (ValidateArrayForManipulation(array))
+            {
+                int[] tempArray = new int[array.Length];
+                for (int i = 0; i < array.Length; i++)
+                {
+                    tempArray[i] = array[i];
+                }
+                for (int i = 0; i < array.Length; i++)
+                {
+                    array[i] = tempArray[array.Length - 1 - i];
+                }
```

Comment:
This is not a request, try to think about it if you like to.
There is no need for the tempArray. :)